tina/customTinaFormFields/textInputWithCount.tsx

```diff
@@ -0,0 +1,28 @@
+//ripped out from SSW's Website 😳
+
+import React from "react";
+import { wrapFieldsWithMeta } from "tinacms";
+
+export const TextInputWithCount = (max: number, isTextArea: boolean = false) =>
+  wrapFieldsWithMeta(({ input }) => (
+    <div className="flex flex-col gap-2">
+      {isTextArea ? (
+        <textarea
+          // eslint-disable-next-line tailwindcss/no-custom-classname
```

Comment:
why are we disabling the eslint here? add comment pls

Reply:
I'll investigate

Reply:
Removed 👌